wiki/PlaceholderExpansion.md

```diff
@@ -1,397 +1,289 @@
+[mathexpansion]: https://github.com/Andre601/Math-expansion
+
+[relational]: https://github.com/PlaceholderAPI/PlaceholderAPI/blob/master/src/main/java/me/clip/placeholderapi/expansion/Relational.java
 
-PlaceholderAPI is using Expansions for its placeholders, with PlaceholderAPI providing the core. Users can download Expansions from the cloud server using commands in-game or by going [here](https://api.extendedclip.com/home/) if they want to use the placeholder.
+## Overview
+This page will cover how you can create your own [`PlaceholderExpansion`][placeholderexpansion] which you can either [[Upload to the eCloud|Expansion cloud]] or integrate into your own plugin.
 
-## Note
-You can either make a separate jar file, to upload it to the expansion-cloud (recommended) or have it as a local class inside your plugin.
+Something to note here is, that PlaceholderAPI relies on Expansions to be installed. PlaceholderAPI is acting as the core while the Expansions are what allows other plugins to use placeholders in various messages.  
+You can download Expansions either directly from the eCloud yourself, or download them through the [[download command of PlaceholderAPI|Commands#papi-ecloud-download]].
 
-## Examples
-There are multiple methods and ways you can use the PlaceholderExpansion.  
-Those depend on what you want to display through the placeholders in the end.
+## Table of Contents
 
-* [Without external plugin](#without-external-plugin)
-* [With external plugin](#with-external-plugin)
-  * [Separate jar](#separate-jar)
-  * [Internal class](#internal-class)
+- [Getting started](#getting-started)
+  - [Common Parts](#common-parts)
+- [Without a Plugin](#without-a-plugin)
+- [With a Plugin (External Jar)](#with-a-plugin-external-jar)
+- [With a Plugin (Internal Jar)](#with-a-plugin-internal-jar)
+  - [Register the Expansion](#register-the-expansion)
+- [Relational Placeholders](#relational-placeholders)
+  - [Notes about Relational Placeholders](#notes-about-relational-placeholders)
 
-### Without an external plugin
-This part here covers how you create an expansion that doesn't require any external/additional plugins to function.  
-Examples of such expansions are:
-- [Player expansion](/PlaceholderAPI/Player-Expansion)
-- [Math expansion](https://github.com/Andre601/Math-Expansion)
-- [Statistics expansion](/PlaceholderAPI/Statistics-Expansion)
+## Getting started
+To get started, first choose what type of [`PlaceholderExpansion`][placeholderexpansion] you want to create. There are various ways you can create a [`PlaceholderExpansion`][placeholderexpansion] from which this page will cover the most common ones.
 
-Since it would be weird (and also make no real sense) to have this inside your plugin, we assume you make a separate jar-file as an expansion.
+### Common Parts
+All shown examples will share the same, common parts that belong to the [`PlaceholderExpansion`][placeholderexpansion] class.  
```

Comment:
```suggestion
All shown examples will share the same common parts that belong to the [`PlaceholderExpansion`][placeholderexpansion] class.
```

Reply:
You removed the two spaces at the end for the line break...